src/scheduler/Calendar/_util.js

```diff
@@ -122,7 +122,7 @@ export const beforeFirstDayInMonth = state => {
 export const afterLastDayInMonth = state => {
   if (Number(state.focusedDayNum) === Number(state.lastDay)) {
     const newMonth = dayjs(state.date)
-      .add(1, "month")
+      .add(1, "day")
```

Comment:
The logic verifies if the last day of the month is the currently selected date. If yes, we either increment the month or not, depending if the last available date was reached.

The logic adds 1 month though to make that check, not merely adding 1 day to get to the next month, i.e. 29 feb will be 29 march, instead of 1st march.

It might worked before though because the last available date setup in the store during initialization was set to 1 month. I set it to 96 hours inadvertently with a copy/paste because this is what is used in Notify. Hence this trigger this bug discovery.

By adding 1 day instead, this should increment to the next month and be within the available time period if it applies.